src/java/org/apache/cassandra/index/sai/disk/SSTableRowIdKeyRangeIterator.java

```diff
@@ -59,17 +64,21 @@ private SSTableRowIdKeyRangeIterator(PrimaryKey min,
                                          long count,
                                          PrimaryKeyMap primaryKeyMap,
                                          QueryContext queryContext,
+                                         SSTableId ssTableId,
                                          RangeIterator<Long> sstableRowIdIterator)
     {
         super(min, max, count);
 
         this.primaryKeyMap = primaryKeyMap;
         this.queryContext = queryContext;
+        // Get and store reference to scores map if it exists.
+        this.ssTableRowIdToScoreMap = queryContext.getScoreCacheForSSTable(ssTableId);
```

Comment:
```suggestion
        this.sstableRowIdToScoreMap = queryContext.getScoreCacheForSSTable(ssTableId);
```

for consistency with the rest